packages/vsphere/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.7.0"
+  changes:
+    - description: Added dimensions field mapping for datastore datastream to support TSDB.
```

```suggestion
    - description: Add dimensions field mapping for datastore datastream to support TSDB.
```

Same reason: https://github.com/elastic/integrations/pull/6333#discussion_r1206407368

---

Also, PR title needs a similar update i.e., s/Added/Add